### ISSUE TITLE
[BUG] SfImage does not allow empty alt text

### ISSUE THREAD

**Issue by rain2o** [vuestorefront/storefront-ui#1652]
**Describe the bug**
The current validator for the SfImage Alt prop does not allow an empty string (e.g. `validator: (value) => value && !!value.trim(),`)
When using decorative images that do not add information to the content, an empty value should be provided to the alt attribute.

**How to reproduce**
1. Use the `<SfImage alt="" />` atom with an empty string in alt prop
2. See errors logged regarding the invalid value

**Expected behavior**
An empty string should be allowed as per the web accessibility regulations. See https://www.w3.org/WAI/tutorials/images/decorative/ 

**Actual behavior**
An error is received -  `Invalid prop: custom validator check failed for prop "alt"`

**Code examples**
Sample code (or your configuration) to help us reproduce the bug faster or to understand your use case.
```
<SfImage
  src="https://via.placeholder.com/150"
  alt=""
  :width="150"
  :height="150"
/>
```

**Additional context**
I can understand the intention to require a value here to try and reinforce accessibility, which is a good thing. However, if you do this, then you need to provide a way to indicate an image is purely decorative so that it allows an empty alt string.

**Comment by mayashavin** [comment]
@rain2o It's not a bug, it's designed to make sure you provide a valid alt text value for accessibility. I'm closing this.

**Comment by mayashavin** [comment]
I will create another improvement ticket to allow either `aria-hidden` or `alt` text to make sure accessibility is preserved.

**Comment by rain2o** [comment]
> @rain2o It's not a bug, it's designed to make sure you provide a valid alt text value for accessibility. I'm closing this.

Because accessibility standards state that you must have an empty alt text for decorative images, having an image component which does not allow empty alt text sounds like a bug.  It is incorrectly enforcing standards.
Either way, bug or improvement, as long as the end result provides the ability to have accessible decorative images with this component, flag it as you like. At the moment the SfImage component is not usable in a front-end which would have a mixture of informative and decorative images.

**Comment by mayashavin** [comment]
@rain2o you are right. Added a fix for it on 0.10.1 hotfix.

**Comment by rain2o** [comment]
Great, thank you @mayashavin !
